2025-04-22 11:51:03,037 INFO    StreamThr :1341656 [internal.py:wandb_internal():85] W&B internal server running at pid: 1341656, started at: 2025-04-22 11:51:03.036882
2025-04-22 11:51:03,040 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: status
2025-04-22 11:51:03,045 INFO    WriterThread:1341656 [datastore.py:open_for_write():87] open: /home/hawk/Desktop/Heavy Detector/Heavy-Clutter-Detection/finalDiffusion/wandb/run-20250422_115103-rei4tyrh/run-rei4tyrh.wandb
2025-04-22 11:51:03,046 DEBUG   SenderThread:1341656 [sender.py:send():379] send: header
2025-04-22 11:51:03,056 DEBUG   SenderThread:1341656 [sender.py:send():379] send: run
2025-04-22 11:51:03,685 INFO    SenderThread:1341656 [dir_watcher.py:__init__():211] watching files in: /home/hawk/Desktop/Heavy Detector/Heavy-Clutter-Detection/finalDiffusion/wandb/run-20250422_115103-rei4tyrh/files
2025-04-22 11:51:03,685 INFO    SenderThread:1341656 [sender.py:_start_run_threads():1188] run started: rei4tyrh with start time 1745311863.036806
2025-04-22 11:51:03,693 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: check_version
2025-04-22 11:51:03,694 DEBUG   SenderThread:1341656 [sender.py:send_request():406] send_request: check_version
2025-04-22 11:51:04,266 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: run_start
2025-04-22 11:51:04,270 DEBUG   HandlerThread:1341656 [system_info.py:__init__():26] System info init
2025-04-22 11:51:04,270 DEBUG   HandlerThread:1341656 [system_info.py:__init__():41] System info init done
2025-04-22 11:51:04,270 INFO    HandlerThread:1341656 [system_monitor.py:start():194] Starting system monitor
2025-04-22 11:51:04,271 INFO    SystemMonitor:1341656 [system_monitor.py:_start():158] Starting system asset monitoring threads
2025-04-22 11:51:04,271 INFO    HandlerThread:1341656 [system_monitor.py:probe():214] Collecting system info
2025-04-22 11:51:04,271 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started cpu monitoring
2025-04-22 11:51:04,271 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started disk monitoring
2025-04-22 11:51:04,272 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started gpu monitoring
2025-04-22 11:51:04,272 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started memory monitoring
2025-04-22 11:51:04,273 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started network monitoring
2025-04-22 11:51:04,282 DEBUG   HandlerThread:1341656 [system_info.py:probe():152] Probing system
2025-04-22 11:51:04,284 DEBUG   HandlerThread:1341656 [system_info.py:_probe_git():137] Probing git
2025-04-22 11:51:04,288 DEBUG   HandlerThread:1341656 [system_info.py:_probe_git():145] Probing git done
2025-04-22 11:51:04,288 DEBUG   HandlerThread:1341656 [system_info.py:probe():200] Probing system done
2025-04-22 11:51:04,288 DEBUG   HandlerThread:1341656 [system_monitor.py:probe():223] {'os': 'Linux-6.8.0-57-generic-x86_64-with-glibc2.35', 'python': '3.10.12', 'heartbeatAt': '2025-04-22T08:51:04.282748', 'startedAt': '2025-04-22T08:51:03.028622', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '<python with no main file>', 'codePathLocal': None, 'git': {'remote': 'https://github.com/arigra/Heavy-Clutter-Detection.git', 'commit': 'ea85cbf106552e8b3107aec6f574024dea911271'}, 'email': 'ari.granevich@gmail.com', 'root': '/home/hawk/Desktop/Heavy Detector/Heavy-Clutter-Detection', 'host': 'hawk-Z790-GAMING-X', 'username': 'hawk', 'executable': '/bin/python3', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 3129.3600000000006, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 1113.725, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1521.238, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 5500.0, 'min': 800.0, 'max': 5600.0}, {'current': 1973.277, 'min': 800.0, 'max': 5600.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5402.626, 'min': 800.0, 'max': 5500.0}, {'current': 1362.777, 'min': 800.0, 'max': 5500.0}, {'current': 4304.081, 'min': 800.0, 'max': 4300.0}, {'current': 4254.5, 'min': 800.0, 'max': 4300.0}, {'current': 4289.743, 'min': 800.0, 'max': 4300.0}, {'current': 4300.022, 'min': 800.0, 'max': 4300.0}, {'current': 4347.222, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 4300.028, 'min': 800.0, 'max': 4300.0}, {'current': 4324.193, 'min': 800.0, 'max': 4300.0}, {'current': 4031.335, 'min': 800.0, 'max': 4300.0}, {'current': 4302.343, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 1832.2072448730469, 'used': 1115.7188682556152}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.62470626831055}}
2025-04-22 11:51:04,288 INFO    HandlerThread:1341656 [system_monitor.py:probe():224] Finished collecting system info
2025-04-22 11:51:04,288 INFO    HandlerThread:1341656 [system_monitor.py:probe():227] Publishing system info
2025-04-22 11:51:04,289 INFO    HandlerThread:1341656 [system_monitor.py:probe():229] Finished publishing system info
2025-04-22 11:51:04,291 DEBUG   SenderThread:1341656 [sender.py:send():379] send: files
2025-04-22 11:51:04,291 INFO    SenderThread:1341656 [sender.py:_save_file():1454] saving file wandb-metadata.json with policy now
2025-04-22 11:51:04,385 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: python_packages
2025-04-22 11:51:04,385 DEBUG   SenderThread:1341656 [sender.py:send_request():406] send_request: python_packages
2025-04-22 11:51:04,425 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: stop_status
2025-04-22 11:51:04,425 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:04,425 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: pause
2025-04-22 11:51:04,425 INFO    HandlerThread:1341656 [handler.py:handle_request_pause():724] stopping system metrics thread
2025-04-22 11:51:04,425 INFO    HandlerThread:1341656 [system_monitor.py:finish():203] Stopping system monitor
2025-04-22 11:51:04,425 DEBUG   SystemMonitor:1341656 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2025-04-22 11:51:04,425 DEBUG   SystemMonitor:1341656 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2025-04-22 11:51:04,425 DEBUG   SystemMonitor:1341656 [system_monitor.py:_start():183] Publishing last batch of metrics
2025-04-22 11:51:04,426 DEBUG   SenderThread:1341656 [sender.py:send_request():406] send_request: stop_status
2025-04-22 11:51:04,427 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined cpu monitor
2025-04-22 11:51:04,427 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined disk monitor
2025-04-22 11:51:04,456 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined gpu monitor
2025-04-22 11:51:04,456 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined memory monitor
2025-04-22 11:51:04,456 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined network monitor
2025-04-22 11:51:04,457 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: resume
2025-04-22 11:51:04,457 INFO    HandlerThread:1341656 [handler.py:handle_request_resume():715] starting system metrics thread
2025-04-22 11:51:04,457 INFO    HandlerThread:1341656 [system_monitor.py:start():194] Starting system monitor
2025-04-22 11:51:04,457 INFO    SystemMonitor:1341656 [system_monitor.py:_start():158] Starting system asset monitoring threads
2025-04-22 11:51:04,457 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started cpu monitoring
2025-04-22 11:51:04,457 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started disk monitoring
2025-04-22 11:51:04,458 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started gpu monitoring
2025-04-22 11:51:04,458 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started memory monitoring
2025-04-22 11:51:04,458 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started network monitoring
2025-04-22 11:51:04,686 INFO    Thread-12 :1341656 [dir_watcher.py:_on_file_created():271] file/dir created: /home/hawk/Desktop/Heavy Detector/Heavy-Clutter-Detection/finalDiffusion/wandb/run-20250422_115103-rei4tyrh/files/requirements.txt
2025-04-22 11:51:04,686 INFO    Thread-12 :1341656 [dir_watcher.py:_on_file_created():271] file/dir created: /home/hawk/Desktop/Heavy Detector/Heavy-Clutter-Detection/finalDiffusion/wandb/run-20250422_115103-rei4tyrh/files/wandb-metadata.json
2025-04-22 11:51:04,822 DEBUG   SenderThread:1341656 [sender.py:send():379] send: telemetry
2025-04-22 11:51:05,343 INFO    wandb-upload_0:1341656 [upload_job.py:push():130] Uploaded file /tmp/tmpszrgc9jawandb/rk81lsyz-wandb-metadata.json
2025-04-22 11:51:05,385 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:06,385 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:07,070 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: pause
2025-04-22 11:51:07,070 INFO    HandlerThread:1341656 [handler.py:handle_request_pause():724] stopping system metrics thread
2025-04-22 11:51:07,070 INFO    HandlerThread:1341656 [system_monitor.py:finish():203] Stopping system monitor
2025-04-22 11:51:07,070 DEBUG   SystemMonitor:1341656 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2025-04-22 11:51:07,070 DEBUG   SystemMonitor:1341656 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2025-04-22 11:51:07,070 DEBUG   SystemMonitor:1341656 [system_monitor.py:_start():183] Publishing last batch of metrics
2025-04-22 11:51:07,070 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined cpu monitor
2025-04-22 11:51:07,071 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined disk monitor
2025-04-22 11:51:07,105 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined gpu monitor
2025-04-22 11:51:07,105 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined memory monitor
2025-04-22 11:51:07,105 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined network monitor
2025-04-22 11:51:07,106 DEBUG   SenderThread:1341656 [sender.py:send():379] send: stats
2025-04-22 11:51:07,111 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: resume
2025-04-22 11:51:07,111 INFO    HandlerThread:1341656 [handler.py:handle_request_resume():715] starting system metrics thread
2025-04-22 11:51:07,111 INFO    HandlerThread:1341656 [system_monitor.py:start():194] Starting system monitor
2025-04-22 11:51:07,111 INFO    SystemMonitor:1341656 [system_monitor.py:_start():158] Starting system asset monitoring threads
2025-04-22 11:51:07,112 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started cpu monitoring
2025-04-22 11:51:07,112 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started disk monitoring
2025-04-22 11:51:07,112 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started gpu monitoring
2025-04-22 11:51:07,113 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started memory monitoring
2025-04-22 11:51:07,113 INFO    SystemMonitor:1341656 [interfaces.py:start():188] Started network monitoring
2025-04-22 11:51:07,389 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:07,691 INFO    Thread-12 :1341656 [dir_watcher.py:_on_file_created():271] file/dir created: /home/hawk/Desktop/Heavy Detector/Heavy-Clutter-Detection/finalDiffusion/wandb/run-20250422_115103-rei4tyrh/files/output.log
2025-04-22 11:51:08,290 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: pause
2025-04-22 11:51:08,290 INFO    HandlerThread:1341656 [handler.py:handle_request_pause():724] stopping system metrics thread
2025-04-22 11:51:08,290 INFO    HandlerThread:1341656 [system_monitor.py:finish():203] Stopping system monitor
2025-04-22 11:51:08,291 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined cpu monitor
2025-04-22 11:51:08,291 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined disk monitor
2025-04-22 11:51:08,291 DEBUG   SystemMonitor:1341656 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2025-04-22 11:51:08,291 DEBUG   SystemMonitor:1341656 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2025-04-22 11:51:08,291 DEBUG   SystemMonitor:1341656 [system_monitor.py:_start():183] Publishing last batch of metrics
2025-04-22 11:51:08,320 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined gpu monitor
2025-04-22 11:51:08,320 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined memory monitor
2025-04-22 11:51:08,320 INFO    HandlerThread:1341656 [interfaces.py:finish():200] Joined network monitor
2025-04-22 11:51:08,320 DEBUG   SenderThread:1341656 [sender.py:send():379] send: stats
2025-04-22 11:51:08,321 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 11:51:08,388 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:09,388 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:09,690 INFO    Thread-12 :1341656 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/hawk/Desktop/Heavy Detector/Heavy-Clutter-Detection/finalDiffusion/wandb/run-20250422_115103-rei4tyrh/files/output.log
2025-04-22 11:51:10,388 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:11,388 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:12,389 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:13,387 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 11:51:13,389 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:14,389 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:15,389 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:16,389 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:17,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:18,388 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 11:51:18,389 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:19,385 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: stop_status
2025-04-22 11:51:19,386 DEBUG   SenderThread:1341656 [sender.py:send_request():406] send_request: stop_status
2025-04-22 11:51:19,427 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:20,389 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:21,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:22,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:23,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:23,597 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 11:51:24,389 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:25,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:26,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:27,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:28,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:29,386 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 11:51:29,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:30,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:31,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:32,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:33,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:34,385 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: stop_status
2025-04-22 11:51:34,386 DEBUG   SenderThread:1341656 [sender.py:send_request():406] send_request: stop_status
2025-04-22 11:51:34,426 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:34,669 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: status_report
2025-04-22 11:51:34,865 ERROR   SenderThread:1341656 [internal_api.py:execute():344] 409 response executing GraphQL.
2025-04-22 11:51:34,865 ERROR   SenderThread:1341656 [internal_api.py:execute():345] {"errors":[{"message":"run rei4tyrh was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2025-04-22 11:51:35,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:36,199 ERROR   SenderThread:1341656 [internal_api.py:execute():344] 409 response executing GraphQL.
2025-04-22 11:51:36,199 ERROR   SenderThread:1341656 [internal_api.py:execute():345] {"errors":[{"message":"run rei4tyrh was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2025-04-22 11:51:36,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:37,390 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:38,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:38,759 ERROR   SenderThread:1341656 [internal_api.py:execute():344] 409 response executing GraphQL.
2025-04-22 11:51:38,759 ERROR   SenderThread:1341656 [internal_api.py:execute():345] {"errors":[{"message":"run rei4tyrh was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2025-04-22 11:51:38,759 INFO    SenderThread:1341656 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 340, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/home/hawk/.local/lib/python3.10/site-packages/requests/models.py", line 1024, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql
2025-04-22 11:51:39,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:40,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:41,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:42,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:43,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:43,535 ERROR   SenderThread:1341656 [internal_api.py:execute():344] 409 response executing GraphQL.
2025-04-22 11:51:43,535 ERROR   SenderThread:1341656 [internal_api.py:execute():345] {"errors":[{"message":"run rei4tyrh was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2025-04-22 11:51:44,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:45,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:46,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:47,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:48,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:49,385 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: stop_status
2025-04-22 11:51:49,426 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:50,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:51,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:52,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:53,196 ERROR   SenderThread:1341656 [internal_api.py:execute():344] 409 response executing GraphQL.
2025-04-22 11:51:53,196 ERROR   SenderThread:1341656 [internal_api.py:execute():345] {"errors":[{"message":"run rei4tyrh was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2025-04-22 11:51:53,391 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:54,392 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:55,392 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:56,392 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:57,392 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:58,392 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:51:59,392 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:00,392 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:01,392 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:02,392 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:03,392 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:04,392 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:05,392 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:06,393 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:07,393 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:08,393 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:09,393 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:10,393 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:10,687 ERROR   SenderThread:1341656 [internal_api.py:execute():344] 409 response executing GraphQL.
2025-04-22 11:52:10,687 ERROR   SenderThread:1341656 [internal_api.py:execute():345] {"errors":[{"message":"run rei4tyrh was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2025-04-22 11:52:11,393 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:12,393 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:13,393 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:14,393 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:15,393 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:16,393 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:17,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:18,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:19,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:20,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:21,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:22,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:23,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:24,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:25,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:26,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:27,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:28,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:29,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:30,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:31,394 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:32,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:33,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:34,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:35,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:36,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:37,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:38,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:39,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:40,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:41,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:42,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:43,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:44,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:45,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:46,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:47,395 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:47,571 ERROR   SenderThread:1341656 [internal_api.py:execute():344] 409 response executing GraphQL.
2025-04-22 11:52:47,571 ERROR   SenderThread:1341656 [internal_api.py:execute():345] {"errors":[{"message":"run rei4tyrh was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2025-04-22 11:52:48,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:49,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:50,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:51,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:52,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:53,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:54,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:55,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:56,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:57,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:58,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:52:59,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:00,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:01,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:02,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:03,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:04,396 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:05,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:06,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:07,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:08,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:09,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:10,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:11,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:12,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:13,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:14,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:15,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:16,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:17,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:18,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:19,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:20,397 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:21,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:22,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:23,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:24,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:25,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:26,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:27,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:28,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:29,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:30,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:31,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:32,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:33,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:34,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:35,398 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:36,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:37,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:38,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:39,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:40,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:41,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:42,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:43,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:44,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:45,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:46,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:47,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:48,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:49,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:50,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:51,399 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:52,400 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:53,400 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:54,400 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:55,400 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:56,400 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:57,400 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:58,400 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:53:59,400 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:54:00,400 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:54:01,400 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:54:02,400 DEBUG   HandlerThread:1341656 [handler.py:handle_request():158] handle_request: internal_messages
2025-04-22 11:54:02,993 ERROR   SenderThread:1341656 [internal_api.py:execute():344] 409 response executing GraphQL.
2025-04-22 11:54:02,993 ERROR   SenderThread:1341656 [internal_api.py:execute():345] {"errors":[{"message":"run rei4tyrh was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2025-04-22 11:54:02,993 INFO    HandlerThread:1341656 [handler.py:finish():882] shutting down handler
2025-04-22 11:54:02,993 INFO    WriterThread:1341656 [datastore.py:close():296] close: /home/hawk/Desktop/Heavy Detector/Heavy-Clutter-Detection/finalDiffusion/wandb/run-20250422_115103-rei4tyrh/run-rei4tyrh.wandb
2025-04-22 11:54:03,220 ERROR   StreamThr :1341656 [internal.py:wandb_internal():173] Thread SenderThread:
Traceback (most recent call last):
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2188, in upsert_run
    response = self.gql(
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 312, in gql
    ret = self._retry_gql(
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 340, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/home/hawk/.local/lib/python3.10/site-packages/requests/models.py", line 1024, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/internal/internal_util.py", line 48, in run
    self._run()
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/internal/internal_util.py", line 92, in _run
    self._debounce()
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/internal/internal.py", line 333, in _debounce
    self._sm.debounce()
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 537, in debounce
    self._maybe_update_config(always=final)
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 514, in _maybe_update_config
    self._debounce_config()
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/sdk/internal/sender.py", line 543, in _debounce_config
    self._api.upsert_run(
  File "/home/hawk/.local/lib/python3.10/site-packages/wandb/apis/normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: run rei4tyrh was previously created and deleted; try a new run name (Error 409: Conflict)
